zerver/views/auth.py

```diff
@@ -158,14 +164,18 @@ def maybe_send_to_registration(
             request.session, "registration_desktop_flow_otp", desktop_flow_otp, expiry_seconds=3600
         )
 
+    multiuse_obj: Optional[MultiuseInvite] = None
+    from_multiuse_invite = False
```

Comment:
It's a bit awkward to move these here, but otherwise mypy complains

Reply:
I think it's about as readable this way as it was the other way, so seems fine.